package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "wallet-ui",
+  "version": "0.1.0",
+  "description": "Wallet UI",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.5.2",
+    "@types/node": "^16.18.76",
+    "@types/react": "^18.2.48",
+    "@types/react-dom": "^18.2.18",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "typescript": "^4.9.5",
```

Comment:
Can we lock these versions? This ensures that when other developers pull down the dependencies for this project, bugs don't creep in via patch releases for essential libraries

Reply:
Locked versions of all library